sympy/physics/control/lti.py

```diff
@@ -3858,26 +3859,24 @@ def state_vector(self, initial_conditions=None, input_vector=None, var=None):
         Examples
```

Comment:
It isn't clear from the docs what happens if initial_conditions or input_vector are left at None. It would helpful to note what the default behavior is.